go/lint-project.sh

```diff
@@ -51,6 +51,17 @@ then
     fi
 fi
 
+# Verify we're using the latest version of github.com/moovfinancial/events if it's a dependency
+eventsLibrary="github.com/moovfinancial/events"
```

Comment:
Can we guard this check to only run in moovfinancial projects? Similar to what https://github.com/moov-io/infra/pull/162/files did.

Reply:
Good idea